dsls/sql/src/main/java/org/apache/beam/dsls/sql/interpreter/operator/functions/BeamSqlFunctionBinaryExpression.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.beam.dsls.sql.interpreter.operator.functions;
+
+import java.util.List;
+import org.apache.beam.dsls.sql.interpreter.operator.BeamSqlExpression;
+import org.apache.beam.dsls.sql.interpreter.operator.BeamSqlPrimitive;
+import org.apache.beam.dsls.sql.schema.BeamSQLRow;
+import org.apache.calcite.sql.type.SqlTypeName;
+
+
+/**
+ * Base class for all binary functions.
+ */
+public abstract class BeamSqlFunctionBinaryExpression extends BeamSqlExpression {
+
+  private BeamSqlFunctionBinaryExpression(List<BeamSqlExpression> operands,
+      SqlTypeName outputType) {
+    super(operands, outputType);
+  }
+
+  public BeamSqlFunctionBinaryExpression(List<BeamSqlExpression> operands) {
+    this(operands, SqlTypeName.ANY);
+  }
+
+  @Override public boolean accept() {
```

Comment:
The logic of  `accept`(only numeric types are permitted) does not name of the class `BeamSqlFunctionBinaryExpression`.